.github/CONTRIBUTING.md

```diff
@@ -154,15 +154,158 @@ the repo such that they can get coding -->
+<!--- Do point to relevant files if you know them, don't just say "there's some css files". 
+- Facts first opinions second. -->
+
+The wording of an issue should be as specific and unambigious as possible. This will help the reader understand exactly what you mean. This can be done in many ways. Therefore we have compiled some examples to demonstrate:
+
+_Example #1: referring to files_ 
+
+If you are referring to a file in the repository do not refer to it as `_typography.css` use the full path instead: `libs/designsystem/src/lib/scss/base/_typography.scss`.
+While it might be obvious for you and experienced contributors, what and where `_typography.css` is; it is not obvious for newcomers. 
+
+If you already have refered to the full path earlier in the issue, it can be okay to refer to the file as `_typography.css`.
+
+_Example #2: writing reproducible steps for bug reports_
+
+The steps should be unambigously specific. This can be done for example by changing the step: 
+> 2. Type some text 
+
+to: 
+
+> 2. Type: 'text'
+ 
+There could be something with the _exact_ way you are doing it, that triggers the behaviour you are experiecing. 
+
+_Example #3: using code snippets to explain what and how_
+
+When reporting a bug or similar do not write something along the lines of: 
+> Whenever i use a `kirby-button` with attention level 1 on the frontpage, a warning is thrown
+
+Specify exactly where and how the button is inserted with a code snippet, along with exact details of the warning and where it can be found:
+> Inserting a `kirby-button` with attention level 1 as shown in the code-snippet, results in the following warning being logged in the browser console: `Warning goes here`.
+
+The latter is pure maintainer poetry. 
+
+These are but three examples of how issues can be specific and unambigious. But always keep in the back of your mind how you can be more specific and unambigious when writing issues. 
+
+#### The good issue follows the template 
+<!-- - Follow the templates! - they are as is to help you remember which information we're looking for -->
+Do not delete the template when creating an issue. Use it. 
+
+The issue templates exists to ease collaboration by providing structure and reminding you of which information other contributors are looking for. 
+They also provide helpful checklists that helps you follow [the process (INSERT PROPER LINK HERE)](#the-process) and lets others know how far you are. 
+
+If you experience that no template lives up to your specific need; well there is a good opportunity to contribute. Create an issue for a new template to be created! 
+
+#### The good issue has a good title
+<!-- - Spend some time on your titles 
+  - Make them alluring. 
+  - Short without too much detail 
+  - The issues are viewed in some sort of list view 80% of the time. -->
+As issues are viewed in some sort of list view 80% of the time (or so) the title is the part being read most often. 
```

Comment:
Consider changing this to 

> The title is the part that is read most often,  as issues are usually viewed in some sort of list.

or something similar

Reply:
> The title of an issue is read most often, as issues are viewed in some sort of list most of the time.  
🤔?

Reply:
Changed it to:
> The title of an issue is the part being read the most, as issues usually are viewed in some sort of list.